lesson_19/page_objects/login_page_pack/login_page.py

```diff
@@ -0,0 +1,81 @@
+from lesson_19.page_objects.login_page_pack.login_page_locators import LoginPageLocators
+from lesson_19.page_objects.main_page_pack.main_page import MainPage
+from lesson_19.utilities.web_ui.base_page import BasePage
+
+
+class LoginPage(BasePage):
+    def __init__(self, driver):
+        super().__init__(driver)
+        self.__page_locator = LoginPageLocators()
+
+    def send_login_email(self, email: str):
+        self.send_keys(locator=self.__page_locator.login_input, value=email)
+        return self
+
+    def send_login_password(self, password: str):
+        self.send_keys(locator=self.__page_locator.password_input, value=password)
+        return self
+
+    def click_login_button(self):
+        self.click(self.__page_locator.sign_in_button)
+        return MainPage(self.driver)
+
+    def login(self, email, password):
+        self.send_login_email(email)
+        self.send_login_password(password)
+        self.click_login_button()
+        return self
```

Comment:
I thought to return Main page, but then I have commented this line because I use method get_password_error_message to check validation text, and if I return return Main page - this method will rase attributeError